<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mFindDialog</class>
 <widget class="QWidget" name="mFindDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>109</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>329</width>
    <height>109</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>329</width>
    <height>109</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>find dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color: rgb(45, 45, 48);
	color: rgb(254, 254, 254);
}

QLineEdit
{
	border-radius: 3px;
	background-color: rgb(51, 51, 55);
	color: rgb(241, 241, 241);
}
</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="mLineEditFind">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Find...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mBtnFind">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Find</string>
         </property>
         <property name="toolTipDuration">
          <number>-1</number>
         </property>
         <property name="whatsThis">
          <string>find button</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../images/images.qrc">
           <normaloff>:/images/Mine/find_replace.png</normaloff>:/images/Mine/find_replace.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mBtnPrevious">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Previous</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../images/images.qrc">
           <normaloff>:/images/themes/default/mActionAtlasPrev.svg</normaloff>:/images/themes/default/mActionAtlasPrev.svg</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mBtnNext">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Next</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../images/images.qrc">
           <normaloff>:/images/themes/default/mActionAtlasNext.svg</normaloff>:/images/themes/default/mActionAtlasNext.svg</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mBtnClose">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>close</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../images/images.qrc">
           <normaloff>:/images/themes/default/mIconClose.svg</normaloff>:/images/themes/default/mIconClose.svg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="mLineEditReplace">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>Replace To...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mBtnReplace">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Replace</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../images/images.qrc">
           <normaloff>:/images/Mine/replace.png</normaloff>:/images/Mine/replace.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mBtnReplaceAll">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>ReplaceAll</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../images/images.qrc">
           <normaloff>:/images/Mine/replaceAll.png</normaloff>:/images/Mine/replaceAll.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="mRadioBtnCaseIntensive">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="toolTip">
          <string>Case insinsibility</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../images/images.qrc">
           <normaloff>:/images/Mine/alphaCase.png</normaloff>:/images/Mine/alphaCase.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
